### ISSUE TITLE
Tensorflow Based Implementation

### ISSUE THREAD

**Issue by arpandaze** [arpandaze/tms-captcha#3]
Previous method of solving the captcha by splitting into strings and counting pixels has low accuracy. This new tensorflow based approach will be hopefully more accurate.

**Tasks:**
- [ ] Create training setup in Python
- [ ] Model loading interface in Typescript
- [ ] Integration with the current code
